src/atoms/BlockLink.js

```diff
@@ -1,9 +1,12 @@
 import styled from 'styled-components';
 
 const BlockLink = styled.a`
+	position: relative;
```

Comment:
Do you really need position relative on all block-level links, or just some of them?